File: javascript-closure-angular-client/API/Client/Transport.js
goog.provide('API.Client.Transport');

/**
 * @record
 */
API.Client.Transport = function() {}

/**
 * Freight model  - 0 - CIF,  - 1 - FOB,  - 2 - Thrid party, - 9 - Free shipping when 9 - Free shipping, the fields below will be ignored if present - 'transporter' - 'withholdICMSTransport' - 'volumes' - 'vehicle' 
 * @type {!string}
 * @export
 */
API.Client.Transport.prototype.modFreight;

/**
 * Forces witholding of ICMS on transport amount (freight)
 * @type {!boolean}
 * @export
 */
API.Client.Transport.prototype.withholdICMSTransport;

/**
 * Packages
 * @type {!Array<!API.Client.Transport_volumes>}
 * @export
 */
API.Client.Transport.prototype.volumes;

/**
 * @type {!API.Client.VehicleTransp}
 * @export
 */
API.Client.Transport.prototype.vehicle;

/** @enum {string} */
API.Client.Transport.ModFreightEnum = { 
  CIF: 'CIF',
  FOB: 'FOB',
  Thridparty: 'Thridparty',
  FreeShipping: 'FreeShipping',
}

File: async-scala-client/src/main/scala/io/swagger/client/model/Transport.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Transport (
  modFreight: String,  // Freight model  - 0 - CIF,  - 1 - FOB,  - 2 - Thrid party, - 9 - Free shipping when 9 - Free shipping, the fields below will be ignored if present - &#39;transporter&#39; - &#39;withholdICMSTransport&#39; - &#39;volumes&#39; - &#39;vehicle&#39; 
withholdICMSTransport: Option[Boolean],  // Forces witholding of ICMS on transport amount (freight)
volumes: Option[List[Transport_volumes]],  // Packages
vehicle: Option[VehicleTransp]
)
